src/chip.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {createUseStyles} from './jss';
+import {useTheme} from './hooks';
+import Box from './box';
+import {Text2} from './text';
+import IconButton from './icon-button';
+import IconCloseRegular from './generated/mistica-icons/icon-close-regular';
+
+const useStyles = createUseStyles(({colors}) => ({
+    container: {
+        display: 'inline-flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        border: `1px solid ${colors.border}`,
+        borderRadius: 16,
+    },
+}));
+
+type ChipProps = {
+    children: string;
+    icon?: React.ReactNode;
+    onClose?: () => void;
+};
+
+const Chip: React.FC<ChipProps> = ({children, icon, onClose}) => {
+    const classes = useStyles();
+    const {colors, texts} = useTheme();
+
+    return (
+        <Box
+            className={classes.container}
+            paddingY={4}
+            paddingLeft={icon ? 8 : 12}
+            paddingRight={onClose ? 8 : 12}
+        >
+            {icon && <Box paddingRight={4}>{icon}</Box>}
+            <Text2 medium>{children}</Text2>
+            {onClose ? (
+                <Box paddingLeft={8}>
+                    <IconButton size={16} aria-label={texts.closeButtonLabel} onPress={() => onClose()}>
+                        <IconCloseRegular size={16} color={colors.neutralHigh} />
+                    </IconButton>
```

Comment:
better make the IconButton size bigger and remove the Box. This way the tappeable are is more accessibe:
```
<IconButton size={24} onPress={...}>
    <IconCloseRegular size={16} />
</IconButton>
```

Reply:
hmm I see that by specs the container has a 6px padding. IMO, this button is too small

Perhaps this could be implemented with smaller paddings and a bigger button

Reply:
> Is it possible to make the tappeable area bigger?
> 
> ```
> <IconButton size={24} onPress={...}>
>     <IconCloseRegular size={16} />
> </IconButton>
> ```

to make that I need to create a custom class to center the icon inside the icon button, but yes, could be done

WDYT about these 2 comments @yceballost ?

Reply:
AFAIK, the icon is automatically centered inside the icon button

Reply:
make sense, can we use a 24px in the bounding box for close action? respecting chips padding? (i think is not possible but I'm open to solutions